README.md

```diff
@@ -106,11 +106,11 @@ shape: (5, 8)
 >>> ## OPTION 1
 >>> # run SQL queries on frame-level
 >>> df.sql("""
-...	SELECT species,
-...	  AVG(sepal_length) AS avg_sepal_length
-...	FROM self
-...	GROUP BY species
-...	""").collect()
+... SELECT species,
```

Comment:
I think this was unintended due to your editor? I don't think we should change this.

Reply:
@ritchie46 what about the `pl.sql()` example at line 131? Shall I align that example with this one while I am at it?